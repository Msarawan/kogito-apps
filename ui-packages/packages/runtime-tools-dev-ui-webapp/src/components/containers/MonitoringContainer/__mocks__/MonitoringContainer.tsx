import React from 'react';

const MockedMonitoringContainer = (): React.ReactElement => {
  return <></>;
};

export default MockedMonitoringContainer;
