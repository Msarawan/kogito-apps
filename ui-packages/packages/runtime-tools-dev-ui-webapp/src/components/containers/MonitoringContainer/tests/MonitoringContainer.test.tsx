import React from 'react';
import { mount } from 'enzyme';
import MonitoringContainer from '../MonitoringContainer';

describe('WebApp - MonitorinContainer tests', () => {
  it('Snapshot test', () => {
    const wrapper = mount(<MonitoringContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
