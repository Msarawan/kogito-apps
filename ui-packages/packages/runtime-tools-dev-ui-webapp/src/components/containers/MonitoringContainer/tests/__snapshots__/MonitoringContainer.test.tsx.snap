// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WebApp - MonitorinContainer tests Snapshot test 1`] = `
<MonitoringContainer>
  <MonitoringView
    dashboard="MonitoringReport.dash.yml"
  >
    <MonitoringWebapp
      dashboard="MonitoringReport.dash.yml"
    >
      <iframe
        src="resources/webapp/monitoring-webapp/index.html?import=MonitoringReport.dash.yml"
        style={
          Object {
            "height": "100%",
            "padding": "10px",
            "width": "100%",
          }
        }
      />
    </MonitoringWebapp>
  </MonitoringView>
</MonitoringContainer>
`;
